scripts/check_ecosystem.py

```diff
@@ -60,39 +60,48 @@ async def clone(self: Self, checkout_dir: Path) -> AsyncIterator[Path]:
             "--no-tags",
         ]
         if self.ref:
-            git_command.extend(["--branch", self.ref])
+            git_clone_command.extend(["--branch", self.ref])
 
-        git_command.extend(
+        git_clone_command.extend(
             [
                 f"https://github.com/{self.org}/{self.repo}",
                 checkout_dir,
             ],
         )
 
-        process = await create_subprocess_exec(
-            *git_command,
+        git_clone_process = await create_subprocess_exec(
+            *git_clone_command,
```

Comment:
I renamed these before extracting `_get_commit` into a separate function. I don't think the extra clarity hurts though.